packages/eslint-config-airbnb-base/rules/style.js

```diff
@@ -3,531 +3,823 @@ module.exports = {
+    // KEEP
     'max-statements': ['off', 10],
 
     // restrict the number of statements per line
     // https://eslint.org/docs/rules/max-statements-per-line
+    // KEEP
     'max-statements-per-line': ['off', { max: 1 }],
 
     // enforce a particular style for multiline comments
     // https://eslint.org/docs/rules/multiline-comment-style
-    'multiline-comment-style': ['off', 'starred-block'],
+    // ENABLE, may need conversation for enforcing JSDoc type comments
+    'multiline-comment-style': ['warn', 'starred-block'],
 
     // require multiline ternary
     // https://eslint.org/docs/rules/multiline-ternary
     // TODO: enable?
+    // KEEP OFF, may warrant a conversation if we want to enforce ternary styling
+    // Essentially, this will enforce whether we want to standardize on
+    //
+    // const value = if a ? b : c
+    //
+    // or
+    //
+    // const value = if a
+    //   ? b
+    //   : c
+    //
+    // or
+    //
+    // const value = if a ?
+    //   b :
+    //   c
+    //
+    // I'm a fan of either leaving this off, or option #2
     'multiline-ternary': ['off', 'never'],
 
     // require a capital letter for constructors
-    'new-cap': ['error', {
-      newIsCap: true,
-      newIsCapExceptions: [],
-      capIsNew: false,
-      capIsNewExceptions: ['Immutable.Map', 'Immutable.Set', 'Immutable.List'],
-    }],
+    // DISABLE: Shouldn't apply to TypeScript
+    'new-cap': [
+      'error',
+      {
+        newIsCap: true,
+        newIsCapExceptions: [],
+        capIsNew: false,
+        capIsNewExceptions: [
+          'Immutable.Map',
+          'Immutable.Set',
+          'Immutable.List',
+        ],
+      },
+    ],
 
     // disallow the omission of parentheses when invoking a constructor with no arguments
     // https://eslint.org/docs/rules/new-parens
+    // KEEP
     'new-parens': 'error',
 
     // allow/disallow an empty newline after var statement
+    // KEEP OFF
     'newline-after-var': 'off',
 
     // https://eslint.org/docs/rules/newline-before-return
+    // KEEP OFF
     'newline-before-return': 'off',
 
     // enforces new line after each method call in the chain to make it
     // more readable and easy to maintain
     // https://eslint.org/docs/rules/newline-per-chained-call
+    // SITUATIONAL ANALYSIS NEEDED:
+    // Personally, I prefer this. What this means is that
+    //
+    // const value: any[] = [].filter(x => x.isThing)
+    //   .map(x => x.value)
+    //   .filter(x => x !== 3)
+    //
+    // is enforced rather than allowing
+    //
+    // const value: any[] = [].filter(x.xIsThing).map(x => x.value).filter(x => x!== 3);
+    //
+    // But this should probably be a conversation, because we have one-line chains throughout the app, and we also need to consider
+    // whether or not chains of RxJS operators within a pipe operator should fall into this convention
     'newline-per-chained-call': ['error', { ignoreChainWithDepth: 4 }],
```

Comment:
SITUATIONAL ANALYSIS NEEDED: Personally, I prefer this. What this means is that
```
const value: any[] = [].filter(x => x.isThing)
  .map(x => x.value)
  .filter(x => x !== 3);
```
is enforced rather than allowing
```
const value: any[] = [].filter(x.xIsThing).map(x => x.value).filter(x => x!== 3);
```
But this should probably be a conversation, because we have one-line chains throughout the app, and we also need to consider whether or not chains of RxJS operators within a pipe operator should fall into this convention.